test/test_jit.py

```diff
@@ -6657,13 +6673,9 @@ def test_passing_unknown_kwargs(self):
 
     def test_passing_and_returning_lists(self):
         # Replace with actual test once we support lists.
-        with self.assertRaisesRegex(
-            RuntimeError,
-            "Lists and strings are not supported yet"
-        ):
-            a, b = torch.ones(5), torch.zeros(5)
-            output = torch.ops.aten.stack([a, b])
-            self.assertEqual(output, torch.ones(10))
+        a, b = torch.ones(5), torch.ones(5)
+        output = torch.ops.aten.cat([a, b])
+        self.assertEqual(output, torch.ones(10))
```

Comment:
Again, would be better to have a test that actually cares that it was `cat([a, b])` and not `cat([b, a])`.